rules/opentitan/defs.bzl

```diff
@@ -175,15 +175,15 @@ def opentitan_test(
                 execution environments.  The values are the kwargs parameter names
                 of the exec_env override or None.  If None, the default parameter
                 names of `cw310`, `dv`, `silicon`, or `verilator` will be guessed.
-      cw310: Execution overrides for a CW310-based test.
+      fpga: Execution overrides for a CW310-based test.
       dv: Execution overrides for a DV-based test.
       silicon: Execution overrides for a silicon-based test.
       verilator: Execution overrides for a verilator-based test.
       kwargs: Additional execution overrides identified by the `exec_env` dict.
     """
     test_parameters = {
-        "cw310": cw310,
-        "cw340": cw340,
+        "cw310": fpga,
+        "fpga": fpga,
```

Comment:
Shall we add a newline for
```
"cw340": fpga
```

Reply:
No, the `cw310` line was included as a temporary measure to prevent existing PRs that have passed their CI from causing breakages when merged. It will be removed once the current set of PRs has cleared.

If you want a different set of parameters for your CW310 and CW340 environments, you can manually include two `fpga_params` and point your execution environments at them by name instead of using `None`.